integreat_cms/cms/views/language_tree/language_tree_node_form_view.py

```diff
@@ -42,3 +43,30 @@ def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
         kwargs["region"] = self.request.region
         return kwargs
+
+
+class LanguageTreeNodeUpdateView(CustomUpdateView):
+    """
+    Class that handles activating/deactivating of a language tree node
+    """
+
+    def form_valid(self, form):
+        response = super().form_valid(form)
+
+        language_tree_node = self.get_object()
+
+        if "active" in form.changed_data:
+            region = language_tree_node.region
+            language = language_tree_node.language
+            models = [PageTranslation, EventTranslation, POITranslation]
+            for model in models:
+                filters = {
+                    f"{model.foreign_field()}__region": region,
+                    "language": language,
+                }
+                for translation in model.objects.filter(**filters):
+                    if self.get_object().active:
```

Comment:
```suggestion
            models = [PageTranslation, EventTranslation, POITranslation]
            for model in models:
                filters = {
                    f"{model.foreign_field()}__region": language_tree_node.region,
                    "language": language_tree_node.language,
                }
                distinct = [f"{model.foreign_field()}__pk", "language__pk"]
                for translation in model.objects.filter(**filters).distinct(*distinct):
                    if language_tree_node.active:
```
I think we can limit the translations to the latest versions of each content type and language...